draft-kazuho-httpbis-priority.md

```diff
@@ -226,6 +226,20 @@ extensible priority scheme when and only when all the requests that are to be
 sent on that backend connection originates from one client-side connection that
 has negotiated the use of the extensible priority scheme (see {{fairness}}).
 
+# Extensible Priorities
+
+This priority design is intended to be extensible, and uses a sequence of
+key-value pairs encoded as a Structured Headers Dictionary
+({{!STRUCTURED-HEADERS}}) to enable extensibility. Each dictionary member
+represents a parameter of the Priority header field.
+
+Care must be taken to ensure any use of extisting parameters are either
```

Comment:
```suggestion
Care must be taken to ensure any use of existing parameters are either
```

Reply:
I think the paragraph would also benefit from starting with the subject e.g. "When attempting to extend priorities, care must be taken.."